upload-api/test/service/ucan-invocation.test.js

```diff
@@ -1,210 +0,0 @@
-import { s3 as test } from '../helpers/context.js'
```

Comment:
All this file content was moved into top level test folder given they are not really within ucanto service.